data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java

```diff
@@ -70,52 +65,36 @@ public Object execute(String[] params) {
 
 	public void persist() {
 		Lineage lineage = new Lineage();
-
-//		LineageQueryInfo LineageQueryInfo = new LineageQueryInfo();
-//		LineageQueryInfo.setProcessId(122);
-//		LineageQueryInfo.setQueryId(UUID.randomUUID().toString());
-//		LineageQueryInfo.setQueryTypeId(1);
-//		LineageQueryInfo.setQueryString("select * from me");
-//		LineageQueryInfo.setInstanceExecId((long)1);
-//		LineageQueryInfo.setCreateTs(new Date());
-//		s.insert("call_procedures.InsertLineageQuery", lineageQueryInfo);
-		lineage.insertLineageQuery(lineageQueryInfo);
-
+		//inserting to LQ
+		//lineage.insertLineageQuery(lineageQueryInfo);
 
 		for (LineageNodeInfo lineageTableNode : lineageTableNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageTableNode);
 			lineage.insertLineageNode(lineageTableNode);
 		}
 		for (LineageNodeInfo lineageColumnNode : lineageColumnNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageColumnNode);
 			lineage.insertLineageNode(lineageColumnNode);
 		}
 		for (LineageNodeInfo lineageFunctionNode : lineageFunctionNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageFunctionNode);
 			lineage.insertLineageNode(lineageFunctionNode);
 		}
 		for (LineageNodeInfo lineageConstantNode : lineageConstantNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageConstantNode);
 			lineage.insertLineageNode(lineageConstantNode);
 		}
-		for (LineageRelationInfo LineageRelationInfo : lineageRelationInfos) {
-//			s.insert("call_procedures.InsertLineageRelation", LineageRelationInfo);
-			lineage.insertLineageRelation(LineageRelationInfo);
-
+		for (LineageRelationInfo lineageRelationInfo : lineageRelationInfos) {
+			lineage.insertLineageRelation(lineageRelationInfo);
 		}
-
 		System.out.println("All nodes and relations are persisted in MySql db");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)